libse/SubtitleFormats/Pac.cs

```diff
@@ -1324,11 +951,11 @@ private int AutoDetectEncoding()
                     {
                         _codePage = CodePageLatin;
                         var sb = new StringBuilder("ABCDEFGHIJKLMNOPPQRSTUVWXYZÆØÅÄÖÜabcdefghijklmnopqrstuvwxyzæøäåü(1234567890, .!?-\r\n'\"):;&");

```

Comment:
Some of the characters here is already present in `LatinCodes.Values`. e.g: `ÆØÅÄÖÜ`...